sequelize/data/user-role.json

```diff
@@ -25,6 +25,11 @@
         "path": "/entity",
         "method": "GET"
     },
+    {
```

Comment:
Since you're wildcarding, do you need the rule above this one?

Reply:
My understanding is that Casbin is touchy. It is very literal, so anything leading up to a wildcard isn't allowed because of the wildcard, but I'll test this.